mono/mini/mini-amd64.c

```diff
@@ -50,6 +50,8 @@
 #include "mini-runtime.h"
 #include "aot-runtime.h"
 
+#define EXTRA_CODE_SPACE (16)
```

Comment:
Is this still needed as the constant was moved to mini.h?

Reply:
I left it a local variable in mini.h but yes we could make it a define there.

Reply:
EXTRA_CODE_SPACE in mini-amd64.c will go away with https://github.com/mono/mono/pull/8437.
We already pad out the base tailcall size to 255, and then with 8437 we can count up the parameters accurately, so the extra 64 seems like overkill upon overkill (the 255 is overkill, the parameter counting only slightly overkill)